src/util.js

```diff
@@ -24,6 +24,20 @@ var Util = {
         total: 0,
         message: null
     },
+    
+    countOccurences: function(contentString, searchString) {
+        var count = 0;
+
+        var searchRegex = new RegExp(searchString, "g");
```

Comment:
Building up a regex dynamically like this is a bit dangerous.

If we only want the user to search for raw strings, building a regex dynamically from their input requires us to sanitize it. [MDN has an escapeRegExp function to do this sanitization if we need it](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions), or I am personally partial to writing the index based string code when doing raw string operations--there's no way to mess up escaping, and it should be faster than dynamically generated regexes.

If we want the users to be able to search using regular expressions, we'll need to wrap this call in a try/catch, since `new RegExp` can throw a `SyntaxError`, for example, if you have a regexp of `[`. The search string count and replacing seem to work when searching for a regex, but the highlighting doesn't.

I think that for our users, raw find/replace without regular expressions is probably what we want, at least to start with.